### PR TITLE
Update hub for desktop guide

### LINKED ISSUE #14635: Set up Desktop Guide content appropriately
Things I've noticed:

- It needs an entry on the main index page
- It doesn't have a special ms.prod/ms.technology value - it's using the default ms.author, author, ms.prod, etc.
- Set up labels and rules

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

- Added Desktop cards for WPF that specify Core/Framework
- Added entry under Core for desktop guide. (Adding another main card for desktop guide on the front page seemed like a bad idea, it creates another row)
- Added ms.tech/prod for desktop-guide in docfx
- Added label process rules

Fixes #14635

### CODE REVIEW

**Comment by Thraka** [comment]
@mairaw I didn't realize you needed to file a specific technology that is allowed with a specific product. (seems kind of silly) so I just filed a ticket to get that approved. I don't see why it wouldn't be.

File: .ghal.rules.json [hunk 1]
```diff
@@ -29,6 +30,9 @@
               "(?i)dotnet-roslyn-api$": ":books: Area - Roslyn API Reference,:file_folder: Repo - roslyn-api-docs",
               "(?i)dotnet-architecture$": ":books: Area - .NET Architecture Guide"
             },
+            "technology": {
+              "(?i)dotnet-wpf$": ":card_file_box: Technology - WPF"
```
Comment (mairaw) [inline .ghal.rules.json]: I'm looking at the approved metadata values and this is not supported

**Review by mairaw — APPROVED** [review]
Other than the metadata issue, everything looks really good!

File: .ghal.rules.json (hunk 1 above)
Comment (Thraka) [reply .ghal.rules.json]: @mairaw 
I filed a ticket yesterday.. I didn't realize that I needed to associate the technology with a service. The service was approved, I don't see why the tech wouldn't be also.

**Comment by mairaw** [comment]
@Thraka if the metadata request has been filed, I think it would be ok to get the warnings for a while.

**Pull request merged**